readthedocs/core/tests/test_homepage.py

```diff
@@ -0,0 +1,44 @@
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+
+class HomepageTest(TestCase):
+
+    def test_homepage_auth(self):
+        user = User(username="user")
+        user.set_password("user")
+        user.save()
+        self.client.login(username="user", password="user")
```

Comment:
Same for the other tests

```suggestion
        user = User(username="user")
        self.client.force_login(user)
```